.github/scripts/create_app_version_pr.sh

```diff
@@ -6,22 +6,26 @@ cd packages/mobile
 # ensure that we are using ssh
 git remote set-url origin git@github.com:valora-inc/wallet.git
 
-app_version="$(node -p "require('./package.json').version")"
-
-echo "Create branch"
+echo "Create version bump branch from main"
 git checkout -b $BRANCH_NAME
 
 echo "Bump app version"
-yarn pre-deploy --minor
+yarn pre-deploy --minor --no-disclaimer
```

Comment:
nit: clearer I think
```suggestion
yarn pre-deploy --minor --no-license-update
```

Reply:
yes you're right!! i've updated both places now :) good idea!